Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
```

Comment:
I don't agree. Users are confused with the current behaviour and I think this would be an improvement. If you want you can make it `HOMEBREW_DEVELOPER` only for now but I do think it's a sensible change to make.